packages/devtools/lib/src/vm_service_wrapper.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:async';
 
+import 'package:devtools/src/profiler/cpu_profile_model.dart';
```

Comment:
make this a relative import. I wish the quick fixes for adding an import defaulted to relative. I've lost track of the number of times I have been bitten by this.

Reply:
Done.